assets/src/data/model/event/index.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+import { isUndefined } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal imports
+ */
+import { getQueryString as baseGetQueryString } from '../base';
+
+export const nowDateAndTime = moment();
+
+/**
+ * Described attributes for this model
+ * @type {{attributes: *}}
+ */
+export const queryDataTypes = {
+	queryData: PropTypes.shape( {
+		limit: PropTypes.number,
+		orderBy: PropTypes.oneOf( [
+			'EVT_name',
+			'EVT_ID',
+			'start_date',
+			'end_date',
+			'ticket_start',
+			'ticket_end',
+		] ),
+		order: PropTypes.oneOf( [ 'asc', 'desc' ] ),
```

Comment:
same question as with the datetimes model... allow for caps, convert case, or use constants ?

Reply:
fixed